crypt/test/data/SteadyStateCrypt/archive/cell_population_sim_at_time_150.arch

```diff
@@ -19,4947 +19,4947 @@
-20 0 0 25 12 1.50000000000000000e+02 4294967295 0 0 0 0 0 5 1 0 1 49 12 CellEdgeData 1 0
```

Comment:
Not quite obvious to me why 
 * (i) this archive has needed to change, 
 * (ii) this alteration from `CellEdgeData` to `CellId` has occurred even if archive needs updating? Is this expected from some other work, and we managed backwards compatible archive loading until now?

Reply:
The archive changed as the member variables of MeshBasedCellPopulation changed so things wont load anymore I don't know why the reordering happened as its archiving magic.

Reply:
Looking a bit more i think the change to the archive (i.e the switch between CellIID ad CellEdgeData) happened earlier but didn't break the loading of the archive so didnt set anything off.  This time non of the numerical values have changed so all is fine. Im actually going to make an issue to discuss the ArchiveFormat as I'm not sure its a worthwhile test anymore ansd may want to look to replcace it. Or make it catch issues sooner.